### PR TITLE
experiment  dao tests

### PR DESCRIPTION
**Pull request by frankie13**

### CODE REVIEW

**Comment by tonydebat** [comment]
Change indents to spaces in SQLExperimentDAOTest.java.
:+1:

**Comment by frankie13** [comment]
@tonydebat formatting sorted.

File: pom.xml [hunk 1]
```diff
@@ -348,7 +348,7 @@
         <version>2.7</version>
         <configuration>
           <formats>
-            <format>xml</format>
+            <format>html</format>
```
Comment (djcooke) [inline pom.xml]: Was this committed intentionally? Is the xml used by Jenkins or anything?

**Comment by frankie13** [comment]
:spaghetti:

File: sqlstore/src/test/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLExperimentDAOTest.java [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+    dao.setDataObjectFactory(new TgacDataObjectFactory());
+  }
+
+  /**
+   * Test method for {@link uk.ac.bbsrc.tgac.miso.sqlstore.SQLExperimentDAO#save(uk.ac.bbsrc.tgac.miso.core.data.Experiment)} .
+   * 
+   * @throws IOException
+   */
+  @Test
+  public void testSave() throws IOException {
```
Comment (djcooke) [inline sqlstore/src/test/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLExperimentDAOTest.java]: Update/edit case of save method not tested

File: pom.xml (hunk 1 above)
Comment (frankie13) [inline pom.xml]: :white_check_mark:

**Comment by djcooke** [comment]
:+1:

**Pull request merged**